src/assets/conclusion.js

```diff
@@ -7,23 +8,23 @@ export default [
       head: `1. Lessening the focus on targeted advertisements at the slight cost of revenue loss`,
       pic: dial,
       alt: `A hand on a dial`,
-      content: `Facebook has decided to rely less on trends in demographic data and more on specific qualities that the advertiser chooses to focus on. This would decrease the advertising algorithm's performance, but lessen the influence of biases within data. However, this still leaves room for the advertiser to impose their biases onto the algorithm, which leads to the next point...`
+      content: <text>Facebook has decided to rely <i>less</i> on trends in demographic data and <i>more</i> on specific qualities that the advertiser chooses to focus on. This would <u>decrease the advertising algorithm's performance, but lessen the influence of biases within data.</u> However, this still leaves room for the advertiser to impose their biases onto the algorithm, which leads to the next point...</text>
```

Comment:
Inside my javascript console, it says that the text tag is unsupported by the browser. If you still want to use semantic HTML tags, maybe use the <p> tag instead everywhere that you use <text> ?

Reply:
looks like markdown screwed that up, I meant to say the p tag